hw/ip/aes/pre_sca/alma/README.md

```diff
@@ -0,0 +1,151 @@
+
+   ```sh
+   ./verify.py --json tmp/circuit.json \
+      --label ${REPO_TOP}/hw/ip/aes/pre_sca/alma/labels/aes_sbox.txt \
+      --top-module aes_sbox \
+      --vcd tmp/tmp.vcd \
+      --rst-name rst_ni --rst-phase 0 \
+      --probe-duration once --mode transient \
+      --glitch-behavior loose --cycles 6
+   ```
+  This should produce output similar to the one below
+  ```sh
+  | CircuitGraph | Total: 1692 | Linear:  504 | Non-linear:  375 | Registers:  167 | Mux:  228 |
+  tmp/tmp.vcd:1286: [WARNING] Entry for name clk_i already exists in namemap (clk_i -> K,)
+  tmp/tmp.vcd:1287: [WARNING] Entry for name data_i already exists in namemap (data_i -> L,)
+  tmp/tmp.vcd:1288: [WARNING] Entry for name data_o already exists in namemap (data_o -> M,)
+  tmp/tmp.vcd:1289: [WARNING] Entry for name en_i already exists in namemap (en_i -> N,)
+  tmp/tmp.vcd:1381: [WARNING] Entry for name mask_i already exists in namemap (mask_i -> O,)
+  tmp/tmp.vcd:1382: [WARNING] Entry for name mask_o already exists in namemap (mask_o -> P,)
+  tmp/tmp.vcd:1383: [WARNING] Entry for name op_i already exists in namemap (op_i -> Q,)
+  tmp/tmp.vcd:1384: [WARNING] Entry for name out_ack_i already exists in namemap (out_ack_i -> R,)
+  tmp/tmp.vcd:1385: [WARNING] Entry for name out_req_o already exists in namemap (out_req_o -> S,)
+  tmp/tmp.vcd:1386: [WARNING] Entry for name prd_i already exists in namemap (prd_i -> T,)
+  tmp/tmp.vcd:1387: [WARNING] Entry for name rst_ni already exists in namemap (rst_ni -> U,)
+  0
+  0
+  Building formula for cycle 0: vars 0 clauses 0
+  Checking cycle 0:
+  Building formula for cycle 1: vars 114 clauses 312
+  Checking cycle 1:
+  Building formula for cycle 2: vars 3276 clauses 10534
+  Checking cycle 2:
+  Checking probe (cycle 2, and _0436_[0]): 0.00
+  Checking probe (cycle 2, and _0378_[0]): 0.00
+  ...
+  Checking probe (cycle 5, not gen_sbox_masked.gen_sbox_dom.u_aes_sbox.u_aes_dom_inverse_gf2p8.u_aes_dom_inverse_gf2p4.u_aes_dom_mul_gamma1_gamma0.u_prim_xilinx_buf_mul_abx_z0.out_o[2]): 0.00           
+  Checking probe (cycle 5, xor gen_sbox_masked.gen_sbox_dom.u_aes_sbox.u_aes_dom_inverse_gf2p8.u_aes_dom_inverse_gf2p4.u_aes_dom_mul_gamma1_gamma0.u_prim_xilinx_buf_mul_abx_z0.out_o[3]): 0.00           
+  Checking probe (cycle 5, mux _0914_[0]): 0.00
+  Checking probe (cycle 5, mux _0912_[0]): 0.00
+  Checking probe (cycle 5, mux _0925_[0]): 0.00
+  Checking probe (cycle 5, mux _0923_[0]): 0.00
+  Checking probe (cycle 5, mux _0945_[0]): 0.00
+  Checking probe (cycle 5, mux _0943_[0]): 0.00
+  Checking probe (cycle 5, mux _0898_[0]): 0.00
+  Checking probe (cycle 5, mux _0913_[0]): 0.00
+  Checking probe (cycle 5, mux _0911_[0]): 0.00
+  Checking probe (cycle 5, mux _0897_[0]): 0.00
+  Finished in 43.31
+  The execution is secure
+  ```
+
+For simplicity, the parse, trace and compile steps can be run with a single script. To this
+end, simply run
+```sh
+${REPO_TOP}/hw/ip/aes/pre_sca/alma/verify_aes.sh
+```
```

Comment:
Maybe it would be worth mentioning that this script exists before giving the detailed steps?

Reply:
Did that, too. Now the how to first explains how to do it with the script and the details are left to the interested user.